packages/npm-dependencies-validation/test/index.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { getDependencyIssues } from "../src/index";
+import { expect, assert } from "chai";
+import * as fsPromises from "fs/promises";
+import { SinonMock, createSandbox } from "sinon";
+
+describe("index unit test", () => {
+  it("2 dependencies and 1 devDependency declared but are not installed", async () => {
+    const result = await getDependencyIssues({
+      fsPath: "./test/projects/no_deps_installed/package.json",
+    });
+    expect(result).to.have.lengthOf(3);
+    // missing json-fixer dependency
+    const jsonFixer = result.find((dep) => dep.name === "json-fixer");
+    assert.isDefined(jsonFixer);
+    expect(jsonFixer?.devDependency).to.be.false;
+    expect(jsonFixer?.type).to.be.equal("missing");
```

Comment:
The connection words should create proper English sentences.

`expect(jsonFixer?.type).to.be.equal("missing");` 

should instead be 
`expect(jsonFixer?.type).to.equal("missing");` 

"expect bamba to equal osem"
"expect bisli to be undefined"